Power BI Tutorial .Rmd

```diff
@@ -0,0 +1,115 @@
+---
+title: "Power BI Tutorial"
+author: "Ellen Chen"
+date: "3/10/2021"
+output:
+  html_document: default
+  pdf_document: default
+---
+
+## Quickstart Tutorial to Microsoft Power BI 
+
+### Preface
```

Comment:
Since main title will be created with single hashtag, all other headings must have 2 hashtags. Follow 2.4.2 (https://jtr13.github.io/cc21/github-submission-instructions.html)